mozilla/blurts-server
# MNTOR-1306: Premium CTA/badge

# References: 
Jira: [MNTOR-1306](https://mozilla-hub.atlassian.net/browse/MNTOR-1306)
Figma: https://www.figma.com/file/iJFOikscJeplRBOuKOtYEW/Onboarding?type=design&mode=design&t=WFSl7cGOyYsWtuTX-0



# Description

Adds the premium upsell CTA for free users and badge for premium users. The upsell dialog will be implemented in [MNTOR-1292](https://mozilla-hub.atlassian.net/browse/MNTOR-1292).

# Screenshot

| CTA | Badge |
| --- | --- |
| <img width="190" alt="mntor-1306-cta" src="https://github.com/mozilla/blurts-server/assets/13835474/39b6dfa0-ebe1-463e-b4b6-c8eec926af95"> | <img width="131" alt="mntor-1306-badge" src="https://github.com/mozilla/blurts-server/assets/13835474/f30017bb-1df9-4118-9362-f18fffad081f"> |

# How to test

- Run Storybook: `npm run storybook`
- You can see the variants under here:
  - CTA: http://localhost:6006/?path=/story/pages-dashboard--dashboard-free-user
  - Badge: http://localhost:6006/?path=/story/pages-dashboard--dashboard-premium-user

# Checklist (Definition of Done)
